demos/src/css/chartAuthoring-demo.css

```diff
@@ -0,0 +1,25 @@
+.floe-ca-pieChart-pie {
+    float: left;
+    width: 400px;
```

Comment:
Thanks to clarify. Yes, the scalable pie issue can be addressed in FLOE-414. Just feel it's a good pratice to always use "em" or "rem" for sizes.